Sources/PackageDescription/LanguageStandardSettings.swift

```diff
@@ -10,166 +10,161 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// The supported C language standard to use for compiling C sources in the package.
-///
-/// Aliases are available for some C language standards. For example,
-/// use `c89`, `c90`, or `iso9899_1990` for the "ISO C 1990" standard.
-/// To learn more, read the [Clang Compiler User's Manual][1].
-///
-/// [1]: <https://clang.llvm.org/docs/UsersManual.html#differences-between-various-standard-modes>
+/// The supported C language standard to use for compiling C sources in the
+/// package.
 public enum CLanguageStandard: String, Encodable {
 
-    /// ISO C 1990.
+    /// The identifier for the C89 language standard.
     case c89
 
-    /// ISO C 1990.
+    /// The identifier for the C90 language standard.
     case c90
 
-    /// ISO C 1999.
+    /// The identifier for the C99 language standard.
     case c99
 
-    /// ISO C 2011.
+    /// The identifier for the C11 language standard.
     case c11
 
-    /// ISO C 2017.
+    /// The identifier for the C17 language stadard.
     @available(_PackageDescription, introduced: 5.4)
     case c17
 
-    /// ISO C 2017.
+    /// The identifier for the C18 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case c18
 
-    /// Working Draft for ISO C2x.
+    /// The identifier for the C2x draft language standard.
     @available(_PackageDescription, introduced: 5.4)
     case c2x
 
-    /// ISO C 1990 with GNU extensions.
+    /// The identifier for the GNU89 language standard.
     case gnu89
 
-    /// ISO C 1990 with GNU extensions.
+    /// The identifier for the GNU90 language standard.
     case gnu90
 
-    /// ISO C 1999 with GNU extensions.
+    /// The identifier for the GNU99 language standard.
     case gnu99
 
-    /// ISO C 2011 with GNU extensions.
+    /// The identifier for the GNU11 language standard.
     case gnu11
 
-    /// ISO C 2017 with GNU extensions.
+    /// The identifier for the GNU17 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnu17
 
-    /// ISO C 2017 with GNU extensions.
+    /// The identifier for the GNU18 language standard.
```

Comment:
Is the change in wording based on the identifier name?  I think the thought was that the identifier name should be brief but the help text should explain the nuance about it being extensions on top of a standard.  Same comment applies to the others in this enum.

Reply:
I see what you mean. This is difficult, because [GNU says](https://gcc.gnu.org/onlinedocs/gcc/C-Dialect-Options.html#C-Dialect-Options) that both `gnu17` and `gnu18` are "GNU dialect of ISO C17." 

Anyway, I was following the pattern of the [currently published documentation](https://developer.apple.com/documentation/swift_packages/clanguagestandard), which was reviewed and approved.

Reply:
This is interesting, because #3079 changed the documentation from what's shown on Apple's documentation page. So, the documentation page is at least outdated by about 16 months.